src/unity/paths/operator_orgs_orgId_limits.yml

```diff
@@ -0,0 +1,70 @@
+get:
+  operationId: getOrganizationLimitsById
+  tags:
+    - Organizations
+    - Operator
+  summary: Get the org's limits
+  parameters:
+    - $ref: "../../common/parameters/TraceSpan.yml"
+    - in: path
+      name: orgId
+      schema:
+        type: string
+      required: true
+      description: The IDPE id of the org
+  responses:
+    '200':
+      description: Organization Limits
+      content:
+        application/json:
+          schema:
+            $ref: '../../quartz/schemas/OrgLimits.yml'
```

Comment:
i didnt want to remake the org limits but i dont think we can move it into common yet because there is are slightly different org limits for idpe. so just referencing them here for now.

Reply:
Out of curiosity, how is the API different for getting the org limits? It looks like our current UI implementation makes a request for an org's limits based on the org_id. If there's no difference, is IDPE or Quartz the source of truth since we should probably reconcile those

Reply:
Quartz gets and returns the org limits pretty much directly from IDPE, but I think we leave out a couple of fields we don't care about.

For the operator UI, we definitely need to request these through Quartz, though, because the operator doesn't have permissions on their own to access org limits for orgs they don't own. We need to use Quartz' operator token to fetch those from IDPE.

Reply:
Aww that makes sense.